### PR TITLE
Di 27909 internal data typed schemas should be push local changes false

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: server-side/rollup.mjs [hunk 1]
```diff
@@ -18,7 +18,7 @@ async function sequentialRollup()
                     format: 'cjs'
                 }
             ],
-            external: ["aws-sdk"],
+            external: ["aws-sdk", "node-fetch"],
```
Comment (Chasky-H) [inline server-side/rollup.mjs:21]: you sure about this?

Comment (saargt) [reply server-side/rollup.mjs:21]: Validating...

Comment (saargt) [reply server-side/rollup.mjs:21]: Seems to work.

Comment (saargt) [reply server-side/rollup.mjs:21]: (Did a quick ADAL POST - ADAL GET check, and it works)

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**